docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: '3.8'
+
+services:
+  eureka-server:
+    container_name: gymcrm-eureka-server-container
+    build:
+      context: ./eureka-server
+    depends_on:
+      - postgres_dev
+      - postgres_prod
+    ports:
+      - "8761:8761"
+
+  report-microservice:
+    container_name: gymcrm-report-microservice-container
+    build:
+      context: ./report-microservice
+    depends_on:
+      - postgres_dev
+      - postgres_prod
+    ports:
+      - "8085:8085"
+
+  main-microservice:
+    container_name: gymcrm-main-microservice-container
+    build:
+      context: ./
+    environment:
+      SPRING_PROFILES_ACTIVE: "dev" # Change to "prod" for production
+
+      # For dev profile:
+      SPRING_DATASOURCE_URL: "jdbc:postgresql://postgres_dev:5432/gymcrm_dev"
+      SPRING_DATASOURCE_USERNAME: "dev_user"
+      SPRING_DATASOURCE_PASSWORD: "dev_1125"
+
+      # For prod profile:
+      # SPRING_DATASOURCE_URL: "jdbc:postgresql://postgres_prod:5432/gymcrm_prod"
+      # SPRING_DATASOURCE_USERNAME: "prod_user"
+      # SPRING_DATASOURCE_PASSWORD: "prod_1125"
+
```

Comment:
There is no need to maintain multiple configurations. One way to save the configuration in system variables, the .env file and take from it. For each environment, system variables will be different. https://docs.docker.com/compose/environment-variables/set-environment-variables/